playwright/tests/organize/people/person/journeys.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { expect } from '@playwright/test';
+
+import ClarasOnboarding from '../../../../mockData/orgs/KPD/journeys/MemberOnboarding/instances/ClarasOnboarding';
+import ClaraZetkin from '../../../../mockData/orgs/KPD/people/ClaraZetkin';
+import KPD from '../../../../mockData/orgs/KPD';
+import MarxistTraining from '../../../../mockData/orgs/KPD/journeys/MarxistTraining';
+import MemberOnboarding from '../../../../mockData/orgs/KPD/journeys/MemberOnboarding';
+import test from '../../../../fixtures/next';
+
+test.describe('Person Profile Page Journeys', () => {
+  test.beforeEach(({ moxy, login }) => {
+    login();
+    moxy.setZetkinApiMock('/orgs/1', 'get', KPD);
+    moxy.setZetkinApiMock(
+      `/orgs/1/people/${ClaraZetkin.id}`,
+      'get',
+      ClaraZetkin
+    );
+
+    moxy.setZetkinApiMock(
+      `/orgs/${KPD.id}/people/${ClaraZetkin.id}/journey_instances`,
+      'get',
+      [ClarasOnboarding]
+    );
+
+    moxy.setZetkinApiMock(`/orgs/${KPD.id}/journeys`, 'get', [
+      MarxistTraining,
+      MemberOnboarding,
+    ]);
+  });
+
+  test.afterEach(({ moxy }) => {
+    moxy.teardown();
+  });
+
+  test('displays relevant journey instances', async ({ appUri, page }) => {
+    await page.goto(appUri + `/organize/${KPD.id}/people/${ClaraZetkin.id}`);
+
+    await expect(
+      page.locator('data-testid=PersonJourneysCard-list')
+    ).toContainText(ClarasOnboarding.title!);
+  });
+
+  test('contains menu to start a journey', async ({ appUri, page }) => {
```

Comment:
This works nicely as a test, but I would suggest that it's unnecessary, and slows down the test overall. The second click and `waitForNavigation` don't need to be done, because you can just inspect the `href` attribute of the element returned from the "`text=${MarxistTraining.title}`"

Would also suggest opening a ticket for a global refactor of Playwright tests wherever a "click and waitForNavigation" is found...